tox.ini

```
@@ -4,6 +4,6 @@ envlist = py27,py35,py36,py37,py38,pypy,pypy3
 [testenv]
 deps =
     setuptools >= 17.1
-    mock <= 3.0.5
+    mock <= 3.0.5; python_version == "2.7"
```

Ooh, neat!  I'd have used a [tox config factor](https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings):

```suggestion
    py27,pypy: mock <= 3.0.5
```

but your way is perhaps cleaner.

It might be nice to have a comment why this pin is needed, but I guess people can git blame and look at git commit messages (which I haven't read because GitHub's review UI makes that awkward).

---

Thanks Marius! I learned something new!